audit/audit-gcp.sh

```diff
@@ -2,20 +2,32 @@
 # set -x -e
 CNCF_GCP_ORG=758905017065
 
+SOLEPROJECT=""
+if [ $# -ge 1 ]
+then
+   SOLEPROJECT="^$1\s"
```

Comment:
what's going on here?

Reply:
Since auditing a project is fairly expensive and time consuming, I tried to add an option to audit just one project at a time. Removed.